From b2c72d04830ee51a2f7d98cc8e49494142307831 Mon Sep 17 00:00:00 2001
Message-Id: <b2c72d04830ee51a2f7d98cc8e49494142307831.1376408877.git.minovotn@redhat.com>
In-Reply-To: <823d94e906b2aeecd2c60bc9f19ca8948a756f27.1376408877.git.minovotn@redhat.com>
References: <823d94e906b2aeecd2c60bc9f19ca8948a756f27.1376408877.git.minovotn@redhat.com>
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Tue, 30 Jul 2013 11:35:26 +0200
Subject: [PATCH 08/18] qemu-char: use ChardevBackendKind in CharDriver

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1375184130-11076-9-git-send-email-kraxel@redhat.com>
Patchwork-id: 52785
O-Subject: [RHEL-6.5 qemu-kvm PATCH 08/12] qemu-char: use ChardevBackendKind in CharDriver
Bugzilla: 676568
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Luiz Capitulino <lcapitulino@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Reviewed-by: Laszlo Ersek <lersek@redhat.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit 99aec0123ce5ba307dc357c5bf1e0f22d2be0931)

Conflicts:

	qemu-char.h
---
 qemu-char-qapi.h |    2 +-
 qemu-char.c      |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-char-qapi.h | 2 +-
 qemu-char.c      | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/qemu-char-qapi.h b/qemu-char-qapi.h
index f971157..c2ecd60 100644
--- a/qemu-char-qapi.h
+++ b/qemu-char-qapi.h
@@ -1,7 +1,7 @@
 #ifndef QEMU_CHAR_QAPI_H
 #define QEMU_CHAR_QAPI_H
 
-void register_char_driver_qapi(const char *name, int kind,
+void register_char_driver_qapi(const char *name, ChardevBackendKind kind,
         void (*parse)(QemuOpts *opts, ChardevBackend *backend, Error **errp));
 CharDriverState *text_console_init(ChardevVC *vc);
 
diff --git a/qemu-char.c b/qemu-char.c
index 131a851..684940c 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -2812,7 +2812,7 @@ typedef struct CharDriver {
     /* old, pre qapi */
     CharDriverState *(*open)(QemuOpts *opts);
     /* new, qapi-based */
-    int kind;
+    ChardevBackendKind kind;
     void (*parse)(QemuOpts *opts, ChardevBackend *backend, Error **errp);
 } CharDriver;
 
@@ -2829,7 +2829,7 @@ void register_char_driver(const char *name, CharDriverState *(*open)(QemuOpts *)
     backends = g_slist_append(backends, s);
 }
 
-void register_char_driver_qapi(const char *name, int kind,
+void register_char_driver_qapi(const char *name, ChardevBackendKind kind,
         void (*parse)(QemuOpts *opts, ChardevBackend *backend, Error **errp))
 {
     CharDriver *s;
-- 
1.7.11.7

